pyproject.toml

```diff
@@ -45,6 +45,7 @@ aiojobs = "^0.3.0"
 Cerberus = "^1.3.4"
 PyYAML = "^5.4.1"
 email-validator = "^1.1.3"
+mock = "^4.0.3"
```

Comment:
1. Remove this. The same functionality is provided by `pytest-mocker`, which is already a development dependency.
2. Poetry distinguishes between development and non-development dependencies. Since `mock` (if we were going to use it) is just used for testing it is a development dependency.
  
   You can flag development dependencies using [`poetry add --dev`](https://python-poetry.org/docs/cli/#add).